policy/approval/approve.go

```diff
@@ -201,13 +201,14 @@ func (r *Rule) IsApproved(ctx context.Context, prctx pull.Context) (bool, string
 	}
 
 	if len(candidates) > 0 && len(approvers) == 0 {
-		msg := fmt.Sprintf("%s needed. Ignored %s from disqualified users",
-			numberOfApprovals(remaining),
+		msg := fmt.Sprintf("%d/%d approvals needed. Ignored %s from disqualified users",
+			len(approvers),
+			r.Requires.Count,
 			numberOfApprovals(len(candidates)))
 		return false, msg, nil
 	}
 
-	msg := fmt.Sprintf("%s needed", numberOfApprovals(remaining))
+	msg := fmt.Sprintf("%d/%d approvals required", len(approvers), r.Requires.Count)
```

Comment:
style: `required` vs. `needed` in the other message. I have a slight preference for "required", but mostly want them to match.